<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>receiver</class>
 <widget class="QDialog" name="receiver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QProgressBar {
      border: 2px solid grey;
      border-radius: 5px;
      text-align: center;
  }
QProgressBar::chunk {
      background-color: #86d8fe;      
	  width: 10px;
      margin: 0.5px;
  }
QLineEdit
{
    background:white;
    padding-left:5px ;
    padding-top:1px ;
    border-bottom-left-radius:3px;
    border-bottom-right-radius:3px;
    border: 1px solid rgb(209 , 209 , 209);
    border-top:transparent;
}
QLineEdit:hover
{
    padding-top:0px ;
    border: 1px solid rgb(21 , 131 , 221);
}</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>进度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>72</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>文件名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>文件大小</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSize">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
